content/tutorials/xrp-ledger-businesses/list-your-exchange-on-xrp-charts.md

```diff
@@ -0,0 +1,199 @@
+
+To enable XRP Charts to list your exchange, complete the following tasks:
+
+1. [Provide a Get Recently Executed Trades REST API endpoint](#provide-a-get-recently-executed-trades-endpoint).
+
+2. [Provide a Get Current Order Book REST API endpoint](#provide-a-get-current-order-book-endpoint).
+
+3. [Send an exchange listing request to XRP Charts](#send-an-exchange-listing-request-to-xrp-charts).
+
+If you have any questions about endpoint specifications, contact xxx@xxxxx.com. ***TODO: do we want to provide a way for them to contact us for dev support?***
+
+
+## Provide a Get Recently Executed Trades Endpoint
+
+Provide a REST API endpoint that returns the most recent 500-1,000 individual trades executed in a particular XRP market.
+
+XRP Charts will query this endpoint every 30 seconds or less, aiming to retrieve responses that contain an overlap in trade data. This will help ensure that XRP Charts does not miss any trades.
+
+***TODO: is the following explanation of how the endpoint will be used - useful or overkill?***
+
+For example, let's say that XRP Charts makes a request to your endpoint and receives trade IDs 1 - 1000 in the response. XRP Charts records data for trade IDs 1-1000. XRP Charts makes a second request 30 seconds later and receives trade IDs 800 - 1800 in the response. There is an overlap in data (and therefore no missed trades), but XRP Charts only records data for the unique trade IDs 1001 - 1800.
+
+Alternatively, let's say that in its second request, XRP Charts receives trade IDs 1500-2500. There is no overlap and XRP Charts has missed some trades; trade IDs 1001-1499, to be exact. In this case, XRP Charts must query your endpoint more often to attempt to receive overlapping data and avoid missing any trades.
+
+
+### Request Format
+
+Provide a request format like the following:
+
+```
+GET {api_base_url}/v1/trades
+```
+
+#### Authentication
+
+We prefer to work with a publicly accessible endpoint.
+
+#### Parameters
+
+Provide parameters that help XRP Charts filter trades returned in the response. Use of these exact parameter field names is optional. While having access to all of these parameters is useful, your endpoint isn't required to provide the ones marked as _(Optional)_.
+
+| Field      | Type    | Description                                           |
+|:-----------|:--------|:------------------------------------------------------|
+| `market`   | String  | Returns trades executed in a particular market in which XRP is either the base or counter currency. Use a suggested format of `<basecurrency>_<countercurrency>` to provide valid values like `xrp_btc`, `btc_xrp`, `xrp_usd`, or `usd_xrp`, for example. |
```

Comment:
Great point. I don't think we can be prescriptive because exchanges with existing endpoints use different formats. For example, Bitso uses `<basecurrency>_<countercurrency>` and Bitstamp uses `<BASECURRENCY><COUNTERCURRENCY>`. I provide the example just so that they can visualize what we are asking for. 

In the same way that we don't want to dictate a parameter field name of `market`, we don't want to dictate valid values for the field. Does that make sense, or am I missing the point? Just let me know! 🙏